bluesky_queueserver/manager.py

```diff
@@ -32,65 +29,107 @@
 """
 
 
-class RunEngineManager:
+class RunEngineManager(Process):
 
-    def __init__(self):
+    def __init__(self, *, conn_watchdog, conn_worker):
```

Comment:
Should this extend rather than replace the argument accepted by `Process`, or else use composition rather than inheritance?